docs/sources/kafka.md

```diff
@@ -0,0 +1,108 @@
+# Kafka event source
+
+This event source acts as a consumer of a Kafka Cluster and forwards all messages it receives
+after wrapping them in a CloudEvent envelope.
+
+## Contents
+
+- [Kafka event source](#kafka-event-source)
+  - [Contents](#contents)
+  - [Prerequisites](#prerequisites)
+  - [Creating a Kafka Source with SASL-PLAIN](#creating-a-kafka-source-with-sasl-plain)
+  - [Creating a Kafka Source with Kerberos-SSL](#creating-a-kafka-source-with-kerberos-ssl)
+    - [Status](#status)
+
+## Prerequisites
+
+A running Kafka cluster.
+
+## Creating a Kafka Source with SASL-PLAIN
+
```

Comment:
A short description here would be useful. Something of the sort, "This section demonstrates how to configure a `KafkaSource` to use SASL-PLAIN authentication."